tests/System.Buffers.Experimental.Tests/ReadOnlyBytesTests.cs

```diff
@@ -372,8 +380,16 @@ public void ReadOnlyBytesEnumeration()
         public void EmptyReadOnlyBytesEnumeration()
         {
             var bytes = ReadOnlyBytes.Empty;
-            Position position = default;
-            Assert.False(bytes.TryGet(ref position, out ReadOnlyMemory<byte> segment));
+            {
```

Comment:
What is the reason for the nested scopes? Is it to re-use `segment` as the variable name?

Reply:
The compiler was complaining about "segment" being redefined even if I don't define it again.

Reply:
That is strange.

Wouldn't something as follows work?
```C#
var bytes = ReadOnlyBytes.Empty;
Position position = default;
Assert.False(bytes.TryGet(ref position, out ReadOnlyMemory<byte> memory));
foreach (var segment in bytes)
{
    Assert.False(true); // As an aside, why is this needed? Can't we check some property on ReadOnlyBytes instead?
}
```